index.js

```diff
@@ -6,12 +6,39 @@ module.exports = {
   extends: ['airbnb'],
   plugins: ['chai-friendly'],
   rules: {
-    'no-unused-expressions': 0,
-    'chai-friendly/no-unused-expressions': 2,
+    'no-unused-expressions': 'off',
+    'chai-friendly/no-unused-expressions': 'error',
     'linebreak-style': ['error', 'unix'],
     'no-param-reassign': ['error', { props: false }],
     'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
     'no-use-before-define': ['error', { functions: false }],
-    'consistent-return': 0,
+    'no-unused-vars': ['error', {'vars': 'all', 'args': 'none'}],
+    'consistent-return': 'off',
+    'arrow-body-style': ['warn', 'as-needed', {
+      requireReturnForObjectLiteral: false,
+    }],
+    'prefer-rest-params': 'warn',
```

Comment:
Why changing this to warn? Rest params are significantly nicer than `arguments`

Reply:
Because this is supported by rest-spread, which i find to be an anti-pattern (lack of fn params)

Reply:
Couldn't the exact same be said of using `arguments`? At least with rest params, you're declaring that the function can take in some arbitrary number of values. And you can take in some named values with it.

Reply:
then why are you using arguments?

Reply:
Because it was done this way in the doc and didn't look for an alternative. If we're ok with removing arguments altogether, fine by me to remove spread & arguments (though it's still used by chai, but eh)

Reply:
I'd be inclined to have this rule as `'error'`, but avoid using them anyway. Then _if_ a need for it comes up, our rule now enforces rest params instead of using `arguments`. If we can `'error'` on both rest params and using `arguments` then I'm fine with that too, as we then make a more conscious decision to use rest params if we feel there's a need.